### PR TITLE
Bug 1232836 - Intermittent pinning_the_web_test.js r=mhenretty

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=694462eb957f737c34da22905fad17d655499f39

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0f1c59adbb2ff68720f95edf24f490a63e667929

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a229d25139086f55d1b0c16812246bb35104109f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4f40b67506bb7c985fb8d9527f89328210dda5e3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dbcb85b0ef3a6148492b82fb578648e9cade2fe5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=70a2eb1dd926cd0fd697ebade08dcc02a6b65422

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bbaeb4a8f91a006e449eeddb569e4a2b55302cde

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f2e05eeb6a51d61ae33cf46e6e9564479cfd25b9

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c800ca114fd8f6a8e258575c2b6b8348181778a5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ba0a1c6de07741fc835825e66fd126de57d69edb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2bd43f8f8a749f030a96d4af6adf1e210b25a8e2

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cbe69e671f6b533c8d456e350e987bc3a1de441c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=20bdf20cc74e311f51c8dfbb217c878ad233ceee

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bc061a8e58c2197fa57b94ed1a57ccb99b221cab

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8987dbca62fb046734d89d4199d08435114c3a05

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e7d4422b7884ff24f733c55c147c445bd21c4cfd

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2f72cac02a9014c1e8abef2a9ecbee13c0fec896

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=21ee5ee2af45a7ecc721b31840eab5bce9bae3d6

File: apps/system/test/marionette/pinning_the_web_test.js [hunk 1]
```diff
@@ -57,7 +57,9 @@ marionette('Pinning the Web', function() {
     var url = server.url('sample.html');
     pinTheWeb.openAndPinSiteFromBrowser(url);
 
-    assert(pinTheWeb.chromeIsPinned());
+    client.waitFor(function() {
+      return pinTheWeb.chromeIsPinned();
+    });
```
Comment (mikehenrty) [inline apps/system/test/marionette/pinning_the_web_test.js]: :+1:

**Pull request merged**